redis/connection.py

```diff
@@ -692,7 +705,11 @@ def on_connect(self):
                 # server seems to be < 6.0.0 which expects a single password
                 # arg. retry auth with just the password.
                 # https://github.com/andymccurdy/redis-py/issues/1274
-                self.send_command("AUTH", self.password, check_health=False)
+                self.send_command(
+                    "AUTH",
+                    auth_args[-1],
+                    check_health=False,
+                )
```

Comment:
Did this really need to be split into multiple lines? `self.password` and `auth_args[-1]` are the same length, so an autoformatter shouldn't complain.